spec/javascript/components/supplemental_application/SupplementalApplicationPage.test.js

```diff
@@ -356,7 +376,11 @@ describe('SupplementalApplicationPage', () => {
 
       // Lease start date
       act(() => {
-        wrapper.find('#form-lease_unit').find('Select').instance().props.onChange({ value: 'id1' })
+        wrapper
+          .find('#form-lease_unit')
+          .find('Select')
+          .instance()
```

Comment:
nit: I know you didn't actually change this part but lets use `.props()` directly instead of `.instance().props`